i18n/states.php

```diff
@@ -698,6 +698,26 @@
 	),
 	'KR' => array(),
 	'KW' => array(),
+	'LA' => array(
+		'laat' => __( 'Attapeu', 'woocommerce' ),
```

Comment:
We can ignore the `la` prefix, also use all in uppercase.

Reply:
Done in 388f15d. Thanks!